src/components/overlayCookie/OverlayCookie.js

```diff
@@ -62,7 +67,7 @@ class OverlayCookie extends Component {
               </span>
             </Trans>
           </p>
-          <p className={styles.cookieIcon}>
+          <p className="text-right my-4">
```

Comment:
One last thing I noticed while deploying to production... Please add `cursor-pointer` to the close icon, so users know that icon is clickable.